### DISCUSSION TITLE
Table's select all checkbox shouldn't select the checkbox which are disabled

### DISCUSSION THREAD

**Discussion by Bhaveshshah777** [primefaces/community#1726] [PrimeNG] [answered]
Suppose I have 8 records, out of them 2 record's check-boxes are disabled. Then by clicking select all even the disabled check-boxes are getting selected. 

Expected Behavior: Program shouldn't select the disabled checkbox when when select all is clicked

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, could you please open an issue with a stackblitz example at primeng repo so we can investigate and address it in the upcoming releases?

Also, please tag me when you opened the issue.

**Reply by Bhaveshshah777** [reply]
Github issue [15338](https://github.com/primefaces/primeng/issues/15338)
